MAINTAINERS.md

```diff
@@ -0,0 +1,61 @@
+# Maintainers
+
+## The maintainer list of all repositories(names are in alphabetical order)
+* Chunlin Yang (Redhat, @clyang82)
+* DONG BEIQING (eBay, @dongbeiqing)
+* Hao Lin (eBay, @haowells)
+* Jian Qiu (Redhat, @qiujian16)
+* Jian Zhu (Redhat, @zhujian7)
+* Le Yang (Redhat, @elgnay)
+* LongLong Cao (Redhat, @morvencao)
+* Mike Ng (Redhat, @mikeshng)
+* Qing Hao (Redhat, @haoqing0110)
+* Zhao Xue (Redhat, @xuezhaojun)
+* Zhiwei Yin (Redhat, @zhiweiyin318)
+
+## The maintainers in different areas of OCM
+
+### [OCM CORE](https://github.com/open-cluster-management-io/ocm)
+* General maintainers:
+    * Jian Qiu
+* Maintainers for specific areas:
+    * `registration`
+        * Le Yang
+        * Wei Liu
+        * Zhao Xue
+    * `work`
+        * Le Yang
+        * Jian Zhu
+    * `placement`
+        * Le Yang
+        * Qing Hao
+    * `operator`
+        * Jian Zhu
+        * Zhiwei Yin
+        * DONG BEIQING
+    * [multicluster-controlplane](https://github.com/open-cluster-management-io/multicluster-controlplane)
```

Comment:
let's move this component into integrations and supported addons section.

Reply:
Moved.